### PR TITLE
Stripe

### PR DESCRIPTION
**Pull request by ahua352**

### CODE REVIEW

File: packages/backend/src/event/event-model.ts [hunk 1]
```diff
@@ -1,5 +1,14 @@
 import { Model, Schema, Types, model } from "mongoose";
 
+/**
+ * A single user registered to an event
+ *
+ * @property {mongoose.Types.ObjectId} userId - the id of the user
+ * @property {Date} registrationDate - the date the user registered for the event
+ * @property {boolean} paymentStatus - whether the user has already paid for the event
```
Comment (JC4github) [inline packages/backend/src/event/event-model.ts]: I believe we decided to change paymentStatus to isPaid

File: packages/backend/src/user/user-model.ts [hunk 2]
```diff
@@ -1,12 +1,24 @@
 import { Model, Schema, Types, model } from "mongoose";
 
+/**
+ * A single event that a User is registered to
+ *
+ * @property {mongoose.Types.ObjectId} eventId - the id of the event
+ * @property {Date} registrationDate - the date the user registered for the event
+ * @property {boolean} paymentStatus - whether the user has already paid for the event
```
Comment (JC4github) [inline packages/backend/src/user/user-model.ts]: For consistency, paymentStatus should be updated to isPaid

File: packages/backend/src/user/user-model.ts [hunk 3]
```diff
@@ -1,12 +1,24 @@
...
+ * @property {object} paymentDetails - the type of payment. For example, "card"
+ *
+ */
 export interface RegistrationRecordUser {
   eventId: Types.ObjectId;
   registrationDate: Date;
-  paymentStatus: string;
+  paymentStatus: boolean;
```
Comment (JC4github) [inline packages/backend/src/user/user-model.ts]: Update paymentStatus to isPaid

File: packages/backend/src/user/user-model.ts [hunk 4]
```diff
@@ -38,15 +51,17 @@ const userSchema = new Schema<IUser>({
   university: String,
   studentId: Number,
   skillLevel: String,
-
   events: [
     new Schema<RegistrationRecordUser>({
       eventId: Types.ObjectId,
       registrationDate: Date,
-      paymentStatus: String,
+      paymentStatus: Boolean,
```
Comment (JC4github) [inline packages/backend/src/user/user-model.ts]: Update paymentStatus to isPaid

**Review by JC4github — CHANGES_REQUESTED** [review]
Just need to refactor paymentStatus to isPiad for consistency, but the rest of the code looks good to me.

**Review by ahua352 — COMMENTED** [review]
I've now renamed paymentStatus to isPaid and merged the webhook branch

**Review by JC4github — APPROVED** [review]
Looks great!

**Pull request merged**